### PR TITLE
[Term Entry] Sdlc: agile

### PR DESCRIPTION
**Pull request by aherman91**
### Description

Add term entry on Agile to SDLC Doc in General. 

### Type of Change

Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/general/concepts/software-development-life-cycle/Terms/agile/agile.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+---
+Title: 'Agile'
+Description: 'Agile development is a software development methodology that emphasizes iteration, flexibility, and cross-team collaboration.'
```
Comment (caupolicandiaz) [inline content/general/concepts/software-development-life-cycle/Terms/agile/agile.md:3]:
```suggestion
Description: 'Agile is a software development methodology that emphasizes iteration, flexibility, and cross-team collaboration.'
```

File: content/general/concepts/software-development-life-cycle/Terms/agile/agile.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  - 'Web Development'
+Tags:
+  - 'Development'
+  - 'Models'
+CatalogContent:
+  - 'learn-how-to-code'
+  - 'paths/full-stack-engineer-career-path'
+---
+
+**Agile Development** is a software development methodology that emphasizes iteration, flexibility and cross-team collaboration.
```
Comment (caupolicandiaz) [inline content/general/concepts/software-development-life-cycle/Terms/agile/agile.md:15]:
```suggestion
**Agile** is a software development methodology that emphasizes iteration, flexibility and cross-team collaboration. 
```

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@aherman91 Looks good overall, just a few notes.

**Comment by aherman91** [comment]
Hey @caupolicandiaz thank you for the feedback! I have made the suggested changes.

**Review by caupolicandiaz — APPROVED** [review]
@aherman91 Looks good.

**Review by HishamT — APPROVED** [review]
LGTM!

**Comment by KTom101** [comment]
I noticed the file path is incorrect, the entry is not appearing on Codecademy. A new PR will need to be created to change `content/general/concepts/software-development-life-cycle/Terms/agile/agile.md` to `content/general/concepts/software-development-life-cycle/terms/agile/agile.md`

**Comment by caupolicandiaz** [comment]
I'm going to add this to one of our link PRs

**Comment by yangc95** [comment]
Thanks @KTom101 for making note of this. I created a PR here which shouldn't alter authorship: https://github.com/Codecademy/docs/pull/1929

cc: @aherman91 @caupolicandiaz @HishamT

**Pull request merged**